kata-deploy/scripts/install-kata-containerd.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+echo "copying kata artifacts onto host"
+cp -R /tmp/kata/* /opt/kata/
+chmod +x /opt/kata/bin/*
+cp /opt/kata/configuration.toml /usr/share/defaults/kata-containers/configuration.toml
+
+## Configure containerd to use Kata:
+echo "create containerd configuration for Kata"
+mkdir -p /etc/containerd/
+
+cat << EOT | tee /etc/containerd/config.toml
+[plugins]
+    [plugins.cri.containerd]
+      snapshotter = "overlayfs"
```

Comment:
what does this do? this would change the behavior for runc too, correct?

Reply:
@jcvenegas - I pulled this from your reference in our CI/guides.  Was this an explicit addition, or can you clarify?

Reply:
Can this be wrapped in a environment variable check? Then those deploying the DaemonSet can skip this behaviour if they customize the config themselves.

Reply:
you can remove it and use the default